student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(name = 'Unknown', age = nil, classroom:, parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] I think it would  be better if you correct the typo error in your constructor method, `classroom:`, right now your app is crushing by showing the below error:
![Screenshot from 2023-08-01 09-02-09](https://github.com/ClintonYounge/School-Libraby-Project/assets/83042742/470d0cbf-a6bd-4f62-b213-d7961f436d98)

You can just update it to something like this:
```ruby
class Student < Person 
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
  ....
end
```